homework-submission/wk1-app/src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import Data from './data.json';
+
+const ListItem = props => {
+  return (
+    <li>
+      {props.description} , {props.deadLine}
+    </li>
+  );
+};
+
+const List = () => {
+  return (
+    <ul>
+      <ListItem description="Get out of bed" deadLine="Wed Sep 13 2017" />
+      <ListItem description="Brush teeth" deadLine="Thu Sep 14 2017" />
+      <ListItem description="Eat breakfast" deadLine="Fri Sep 15 2017" />
+    </ul>
+  );
+};
+
+const Item = Data.map(elem => {
+  return (
+    <li
+      key={elem.id}
+      style={elem.done === true ? { textDecoration: 'line-through' } : { textDecoration: '' }}
+    >
```

Comment:
it is not good to style in 
it is not recommended to make a direct style, better to create a class in CSS file and use it here.
for example:
make a class redColor in your CSS file and to use it in your JS file like:
< li className={`item ${ done && 'redColor' }` }  .. .. ..>

Reply:
thank you @jalal75 for feedback